google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java

```diff
@@ -169,7 +169,7 @@ private TranslateOptions(Builder builder) {
       logger.log(
           Level.WARNING,
           String.format(
-              "Ignoring Application Default Credentials: using explicit setting for API key instead.",
+              "Ignoring Application Default Credentials %s: using explicit setting for API key instead.",
```

Comment:
Hmm. Can we instead do
```java
logger.log(Level.WARNING, "Ignoring Application Default Credentials {}:...", ServiceOptions.CREDENTIAL_ENV_NAME);
```
? That way we don't need to format the string if the log is turned off.

Reply:
Done.